accepted/2021/statics-in-interfaces/README.md

```diff
@@ -1261,31 +1679,369 @@ namespace System
+        // * INumber
+        //   * TSelf Abs(TSelf)
+        //   * TSelf Clamp(TSelf, TSelf, TSelf)
+        //   * TSelf CopySign(TSelf, TSelf)
+        //   * TSelf CreateChecked(TOther)
+        //   * TSelf CreateSaturating(TOther)
+        //   * TSelf CreateTruncating(TOther)
+        //   * bool IsNegative(TSelf)
+        //   * TSelf Max(TSelf, TSelf)
+        //   * TSelf MaxMagnitude(TSelf, TSelf)
+        //   * TSelf Min(TSelf, TSelf)
+        //   * TSelf MinMagnitude(TSelf, TSelf)
+        //   * TSelf Parse(string, NumberStyles, IFormatProvider?)                          // * Existing
+        //   * TSelf Parse(ReadOnlySpan<char>, NumberStyles, IFormatProvider?)              // * Existing
+        //   * int Sign(TSelf)
+        //   * bool TryCreate(TOther, out TSelf)
+        //   * bool TryParse(string?, NumberStyles, IFormatProvider?, out TSelf)            // * Existing
+        //   * bool TryParse(ReadOnlySpan<char>, NumberStyles, IFormatProvider?, out TSelf) // * Existing
+        // * IParsable
+        //   * TSelf Parse(string, IFormatProvider?)                                        // * Existing
+        //   * bool TryParse(string?, IFormatProvider?, out TSelf)
+        // * ISpanFormattable
+        //   * bool TryFormat(Span<char>, out int, ReadOnlySpan<char>, IFormatProvider?)    // * Existing
+        // * ISpanParsable
+        //   * TSelf Parse(ReadOnlySpan<char>, IFormatProvider?)
+        //   * bool TryParse(ReadOnlySpan<char>, IFormatProvider?, out TSelf)
+        // * ISubtractionOperators
+        //   * TSelf operator -(TSelf, TSelf)                                               // * Existing
+        //   * TSelf operator checked -(TSelf, TSelf)                                       // ? Explicit
+        // * IUnaryNegationOperators
+        //   * TSelf operator -(TSelf)                                                      // * Existing
+        //   * TSelf operator checked -(TSelf)                                              // ? Explicit
+        // * IUnaryPlusOperators
+        //   * TSelf operator +(TSelf)                                                      // * Existing
     }
 
     public struct Double
-        : IBinaryFloatingPoint<double>,
+        : IBinaryFloatingPointIeee754<double>,
           IConvertible,
           IMinMaxValue<double>
     {
+        public const double AdditiveIdentity = 0;                                           // ? Expose
+        public const double E = Math.E;
+        public const double MaxValue = 1.7976931348623157E+308;                             // Existing
+        public const double MinValue = -1.7976931348623157E+308;                            // Existing
+        public const double MultiplicativeIdentity = 1;                                     // ? Expose
+        public const double NegativeOne = -1                                                // ? Expose
```

Comment:
Missing semicolon.

Reply:
Thanks, fixed! Shouldn't be too problematic overall. A lot of this is just "short notes" to help simplify API review.